test-core-development.cfg

```diff
@@ -0,0 +1,10 @@
+[buildout]
+extends =
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-package.cfg
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-versions.cfg
```

Comment:
> If we update the Plone version in http://kgs.4teamwork.ch/release/opengever/develop, we'll also need to remember to bump it here. Maybe a big fat comment in the develop KGS would be in order?

@lukasgraf I have updated (a323110) the `test-core-development.cfg` so that it does no longer include the Plone version (same setup as `test-policy.cfg`). That means only the KGS sets the Plone version.

By no longer extending from [`ftw.buildout`'s `test-plone-4.3.2.cfg`](https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-plone-4.3.2.cfg), we no longer extend the general settings automatically, which is ok, I think:
<img width="1552" alt="bildschirmfoto 2017-01-23 um 11 48 17" src="https://cloud.githubusercontent.com/assets/7469/22201175/c7a86896-e162-11e6-82ad-60217264c4bc.png">


However, I realized that we are not including Hotfixes automatically in the tests. 
The KGS extends https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/hotfixes/4.3.2.cfg , but this only defines `hotfix-eggs`, but this does not seem to be used in tests, at least with my changes 😯 

How should I address this? Shouldn't we just include all `hotfix-eggs` (from KGS=) in the test-eggs?